src/NorthwindWebsite/Configuration/MiddlewareConfiguration.cs

```diff
@@ -24,6 +24,8 @@ public static class MiddlewareConfiguration
             app.UseDeveloperExceptionPage();
         }
 
+        app.UseCors("AllowCors");
```

Comment:
Since this is a policy name, let's move it to constant and also add postfix `Policy`.
BTW, have you tried [this](https://github.com/dotnet-fizzyy/ASP.Net-Identity/blob/master/src/IdentityWebApi/Startup/Startup.cs#L84) implementation?